pages/linux/pkgctl.md

```diff
@@ -0,0 +1,8 @@
+# pkgctl
+
+> Unified command-line devtools frontend for Arch Linux.
+> More information: <https://man.archlinux.org/man/pkgctl.1>.
+
+- Download PKGBUILD of a package:
```

Comment:
Does it create a new directory or places `PKGBUILD` etc in the directory in which it was invoked ?

Reply:
I think we might as well add that, what do you think ?

Reply:
There. That should be good.

Reply:
Thank you :)